src/server/test/web/readingsLineMeterRangeRaw.js

```diff
@@ -26,7 +27,70 @@ mocha.describe('readings API', () => {
             mocha.describe('for range (min/max)', () => {
                 mocha.describe('for raw meters', () => {
 
-                    // Add LR9 here
+                    mocha.it('LR9: line chart, range aggregation, raw data (Degrees -> C), daily, no normalization', async () => {
```

Comment:
Sorry to be picky about this but the design document has "range should have daily points for 15 minute reading intervals and raw units with +-inf start/end time & Celsius as Celsius" for the description of LR9. OED wants it to be that to easily find the tests and to keep the strings consistent. The "LR9: " at the start should remain.